src/core/DataBase_List.cpp

```diff
@@ -45,7 +80,27 @@ string List::rpop(string key) {
     return s;
 }
 
-int List::lpush(string key, string value){
+bool DataBase<string, deque<string>>::set(const string& key, const string& value, initializer_list<string> options){
+    if(options.size == 2 && options[0] == 'x'){
```

Comment:
The initializer_list size is a method, not an instance variable, please change it to a method.